### DISCUSSION TITLE
Querying on one-to-many relationship fields with conditional logic

### DISCUSSION THREAD

**Discussion by yooyea** [pocketbase/pocketbase#4663] [Q&A] [answered]
I'm trying to query a one-to-many relationship field in Pocketbase, specifically, I want to filter posts by tags where `tags.tag_name` equals 'aaa' AND `tags.tag_name` equals 'bbb'. I'm not sure how to construct the query to achieve this using Pocketbase's API or query builder. Can someone provide guidance on how to perform this type of conditional query on related fields?

**Comment by yooyea** [comment]
- Querying with `tags.tag_name ?= 'aaa'` returns records as expected.
- Querying with `tags.tag_name ?= 'bbb'` also returns records as expected.
- However, when I try to query with `tags.tag_name ?= 'aaa' && tags.tag_name ?= 'bbb'`, no records are returned, despite the fact that I know there are records in the database that have both 'aaa' and 'bbb' tags

**Reply by ganigeorgiev** [reply, marked as answer]
Please check https://github.com/pocketbase/pocketbase/discussions/4319.

**Reply by yooyea** [reply]
I have tried the search method with 'id', and it worked for me! That's great. Thank you so much for helping me solve the issue.
